amlb/results.py

```diff
@@ -357,8 +357,7 @@ def acc(self):
         return float(accuracy_score(self.truth, self.predictions))
 
     def balacc(self):
-        # return float(balanced_accuracy_score(self.truth, self.predictions))
-        pass
+        return float(balanced_accuracy_score(self.truth, self.predictions))
```

Comment:
yes, you can enable now that the `sklearn` version is supporting this.

> I did not add it to the config but I would like to ask your opinion on that.

I don't think it's necessary to add it to the default `config.yaml`.
What I should add there though, is the list of all supported metrics, as a comment to each `benchmark.metrics` entry, so that users can decide which metrics they can use:
1. as an objective function  for the automl frameworks.
2. as a result metric to compare the predictions.
I also need to improve documentation regarding those metrics, here is how it works:
```
metrics: 
    binary: ['auc', 'logloss', 'acc']`
    multiclass: ['logloss', 'acc']
    regression: ['rmse', 'r2']
```
means that the objective functions given to the frameworks are respectively `auc`, `logloss` and `rmse` (the first of each list).
Those objective metrics are also the metric used for the `result` metric column.
But ALL listed metrics (for a given problem) are computed from the predictions and available as additional information to compare the results.

> Personally, i think balanced accuracy should be enabled. This means we have to tweak the frameworks for support on that.

I don't know if many frameworks will support `balanced accuracy` as an objective function, most likely some of them would need to fallback to another metric, probably `accuracy`.
I just know that H2O's equivalent as an objective function (leading to same behaviour at least) would be `mean_per_class_error`.

Reply:
understood, here I just enable it so that the users can overwrite the metrics through there config file (and balanced accuracy is an option, not I don't think it is the case)!